<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistroyTrending</class>
 <widget class="QWidget" name="HistroyTrending">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1005</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="leftWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>154</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#leftWidget{
background:rgba(81,149,202,1);
border-style:solid;
border-width:0px 16px 0px 0px;
border-color:#447CA7;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="rightWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#rightWidget{
	background:rgba(241,246,255,1);
}</string>
     </property>
     <widget class="QWidget" name="topWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1766</width>
        <height>50</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1766</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#topWidget{
	background:rgba(241,246,255,1);
	border-style:solid;
	border-color:#DFDFDF;
	border-width: 0px 0px 1px 0px;
}</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>15</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">width:64px;
height:21px;
font-size:16px;
font-family:微软雅黑;
color:rgba(38,38,38,1);
line-height:21px;</string>
       </property>
       <property name="text">
        <string>历史趋势</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>1766</width>
        <height>995</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget{
background:#FFFFFF;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>14</number>
         </property>
         <item>
          <widget class="QLineEdit" name="searchEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="searchBtn">
           <property name="text">
            <string>搜索</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>时间选择：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dateCombo">
           <item>
            <property name="text">
             <string>最近一天</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="anyTimeBox">
           <property name="text">
            <string>任意时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit1"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>至</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit2"/>
         </item>
         <item>
          <widget class="QPushButton" name="queryBtn">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="autoCheck">
           <property name="text">
            <string>自动刷新</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="chartArea" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
